app/client/src/features/events/EventLive.tsx

```diff
@@ -64,6 +72,15 @@ export const EVENT_LIVE_QUERY = graphql`
     }
 `;
 
+export const BROADCAST_MESSAGE_MUTATION = graphql`
+    mutation EventLiveMutation($input: CreateBroadcastMessage!) {
+        createBroadcastMessage(input: $input) {
+            isError
+            message
+        }
+    }
+`
+
```

Comment:
Please move all this to a separate component like `useBroadcastMessageMutation`. This component is already quite large and adding it in here makes reading the code harder. It's fine to prototype like this but it's good to get in the habit of separating out from the start.